src/Common/Factory/FactoryCreateContainOneNewRule.php

```diff
@@ -20,10 +21,17 @@
  */
 class FactoryCreateContainOneNewRule extends AbstractFactoryRule implements MethodAware
 {
+    use PhpDocTrait;
+
+    /**
+     * @var string
+     */
+    protected const PATTERN_QUERY_CLASS_NAME = '#Orm\\\\Zed\\\\[A-Za-z]+\\\\Persistence\\\\Spy[A-Za-z]+Query#';
```

Comment:
are we ok that this rule will fail for projects that adds their own tables not in the Spy namespace?

Reply:
Indeed, that regex is way too specific now